src/Output/UnicodeOutput.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PChess\Chess\Output;
+
+use PChess\Chess\Board;
+use PChess\Chess\Chess;
+
+final class UnicodeOutput extends BasicOutput implements OutputInterface
```

Comment:
maybe emoji output :chess_pawn:

Reply:
unfortunately, the only piece present in emoji set is pawn.